src/NUnitFramework/FakeTestMethod.cs

```diff
@@ -0,0 +1,49 @@
+﻿// ***********************************************************************
+// Copyright (c) 2014 Charlie Poole, Rob Prouse
```

Comment:
Trivial - but wrong date here. 🙂

Reply:
That's the year the code was created in the old `Fakes.cs` file. Splitting it into a separate file without change didn't seem like a reason to update the year.